tool/jt.rb

```diff
@@ -517,6 +518,26 @@ def dis(file)
     puts Pathname(file).sub_ext('.ll')
   end
 
+  def environment
+    puts "Environment"
+    env_vars = ['JAVA_HOME','RUBY_BIN','GRAALVM_BIN','GRAAL_HOME',
+    'JVMCI_BIN','JVMCI_GRAAL_HOME','GRAAL_JS_JAR',
+    'SL_JAR','OPENSSL_PREFIX','AOT_BIN','TRUFFLERUBY_CEXT_ENABLED',
+    'TRUFFLERUBYOPT','RUBYOPT']
+    env_vars.each do |e|
+      puts "#{e}: #{ENV[e]}"
+    end
+    raw_sh('ruby -v', continue_on_failure: true)
+    raw_sh('uname -a', continue_on_failure: true)
+    raw_sh('clang -v', continue_on_failure: true)
+    raw_sh('opt -version', continue_on_failure: true)
+    raw_sh('mx version', continue_on_failure: true)
+    sh('mx sversions', continue_on_failure: true)
+    raw_sh('git show -s --format=%H', continue_on_failure: true)
+    raw_sh("#{ENV['OPENSSL_PREFIX']}/bin/openssl version", continue_on_failure: true) 
```

Comment:
This should also work if OPENSSL_PREFIX is not set (e.g. on Linux)

Reply:
Fixed